packages/maskbook/src/web3/constants/index.ts

```diff
@@ -1,5 +1,8 @@
 import { ChainId } from '../types'
 
+export const GAS_CUSTOM_WAIT = 5
```

Comment:
Please comment the unit out.

Reply:
This is the wait time of custom gas price

Reply:
5 minutes or 5 seconds?

Reply:
5 minutes

Reply:
Please leave a comment.